### ISSUE TITLE
CSV ArtifactViewer whites out when clicking a square

### ISSUE THREAD

**Issue by nabenabe0928** [optuna/optuna-dashboard#911]
### Description

When I upload [this CSV file](https://github.com/user-attachments/files/16398586/demo-dataset.csv) as an artifact to a study and click the square (see Figure 1) at the artifact, it whites out.

![image](https://github.com/user-attachments/assets/2297d94c-cbb8-4005-9994-9460c3b839c6)
Figure 1. The icon of the artifact

![image](https://github.com/user-attachments/assets/f0727728-751c-4faf-8451-1e77d8242fcf)
Figure 2. The dashboard page

The page in Figure 2. whites out when clicking the square button.
The URL does not change, the `Go Back` button does not resolve the problem. Refreshing the page fixes the problem.

### How to Reproduce

1. Start dashboard
2. Upload [this CSV file](https://github.com/user-attachments/files/16398586/demo-dataset.csv) to the dashboard
4. Click the button
5. White out

### Python version

3.9

### Optuna version

4.0.0b0

### optuna-dashboard version or git revision

The  HEAD of the main branch 

### Web browser

Google Chrome

**Comment by c-bata** [comment]
@keisuke-umezawa Could you take a look at this issue?

**Comment by c-bata** [comment]
I also notice an another issue that `TableArtifactViewer` is not enabled when using trial artifacts.
https://github.com/optuna/optuna-dashboard/pull/877

**Comment by keisuke-umezawa** [comment]
@c-bata ok! I will handle it.

**Comment by nabenabe0928** [comment]
@keisuke-umezawa @c-bata 

Replacing an empty string with a random string resolved the issue:

```tsx
    return keys.map((key) => ({
      header: key || " ",
      accessorFn: (info: Data) =>
        typeof info[key] === "object" ? JSON.stringify(info[key]) : info[key],
      enableSorting: true,
      enableColumnFilter: false,
    }))
```

https://github.com/optuna/optuna-dashboard/blob/main/optuna_dashboard/ts/components/Artifact/TableArtifactViewer.tsx#L58

**Comment by c-bata** [comment]
Fixed in #913.
